content/guides/models/track/config.md

```diff
@@ -163,38 +164,37 @@ config = {
 }
 
 # Pass the config dictionary when you initialize W&B
-run = wandb.init(project="config_example", config=config)
-
-# Update config after you initialize W&B
-wandb.config["dropout"] = 0.2
-wandb.config.epochs = 4
-wandb.config["batch_size"] = 32
+with wandb.init(project="config_example", config=config) as run:
+    # Update config after you initialize W&B
+    run.config["dropout"] = 0.2
+    run.config.epochs = 4
+    run.config["batch_size"] = 32
 ```
+
 You can update multiple values at a time:
 
 ```python
-wandb.init(config={"epochs": 4, "batch_size": 32})
-# later
-wandb.config.update({"lr": 0.1, "channels": 16})
+run.config.update({"lr": 0.1, "channels": 16})
 ```
 
 ### Set the configuration after your Run has finished
 Use the [W&B Public API]({{< relref "/ref/python/public-api/" >}}) to update your config (or anything else about from a complete Run) after your Run. This is particularly useful if you forgot to log a value during a Run. 
 
-Provide your `entity`, `project name`, and the `Run ID` to update your configuration after a Run has finished. Find these values directly from the Run object itself `wandb.run` or from the [W&B App UI]({{< relref "/guides/models/track/workspaces.md" >}}):
+Provide your `entity`, `project name`, and the `Run ID` to update your configuration after a Run has finished. Find these values directly from the Run object itself or from the [W&B App UI]({{< relref "/guides/models/track/workspaces.md" >}}):
 
 ```python
-api = wandb.Api()
-
-# Access attributes directly from the run object
-# or from the W&B App
-username = wandb.run.entity
-project = wandb.run.project
-run_id = wandb.run.id
-
-run = api.run(f"{username}/{project}/{run_id}")
-run.config["bar"] = 32
-run.update()
+with wandb.init() as run:
+    api = wandb.Api()
+
+    # You can access attributes on the run object directly.
+    username = run.entity
+    project = run.project
+    run_id = run.id
+
+    # NOTE: api.run returns a different type of object than wandb.init().
+    api_run = api.run(f"{username}/{project}/{run_id}")
+    api_run.config["bar"] = 32
+    api_run.update()
```

Comment:
this example seems wrong. based on the text above?

Reply:
can you clarify?

Reply:
oh yeah sure, that we have the context manager to create a run, but the comment above explains how to modify the data post run.

> Provide your `entity`, `project name`, and the `Run ID` to update your configuration after a Run has finished

Reply:
updated!